### ISSUE TITLE
TypeError: Cannot read property '1' of null

### ISSUE THREAD

**Issue by astronomersiva** [beautify-web/js-beautify#1481]
Stack trace:

I am getting the following error in rc5.

```
TypeError: Cannot read property '1' of null
    at Parser.get_tag (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/src/html/beautifier.js:276:60)
    at Beautifier.beautify (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/src/html/beautifier.js:678:37)
    at style_html (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/src/html/index.js:33:21)
    at Function.style_html (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/src/index.js:36:10)
    at makePretty (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/lib/cli.js:448:36)
    at Object.processInputSync (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/lib/cli.js:442:5)
    at Array.forEach (<anonymous>)
    at Object.exports.interpret (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/lib/cli.js:305:15)
    at Object.<anonymous> (/Users/siva/.nvm/versions/node/v8.11.3/lib/node_modules/js-beautify/js/bin/js-beautify.js:4:5)
    at Module._compile (module.js:652:30)
```

This happens when there is a space between the HBS opening tag and the attributes:

`{{ myHelper someValue}}`

## Environment
OS: macOS

## Settings
Example:
```
js-beautify -r -s=2 --wrap_line_length=150 --type=html --indent_handlebars fileName
```

**Comment by bitwiseman** [comment]
Thanks!

**Comment by bitwiseman** [comment]
Fixed in and released in 1.8.0-rc6
